appinventor/blocklyeditor/src/workspace_svg.js

```diff
@@ -36,6 +36,13 @@ Blockly.WorkspaceSvg.prototype.backpack_ = null;
  */
 Blockly.WorkspaceSvg.prototype.componentDb_ = null;
 
+/**
+ * The list of screen names currently in this project.
+ * @type {!Array<string>}
+ * @private
+ */
+Blockly.WorkspaceSvg.prototype.screenList_ = [];
```

Comment:
The array here gets shared by all instances of Blockly.WorkspaceSvg. You may want to initialize this field in Blockly.Blocklyeditor.create instead so that it is per-workspace.